tools/harness-thci/OpenThread.py

```diff
@@ -3492,14 +3492,16 @@ def stopListeningToAddr(self, sAddr):
         return True
 
     @API
-    def registerMulticast(self, sAddr='ff04::1234:777a:1', timeout=MLR_TIMEOUT_MIN):
+    def registerMulticast(self, listAddr=('ff04::1234:777a:1',), timeout=MLR_TIMEOUT_MIN):
```

Comment:
Do we not support the case where `listAddr` is a single multicast address (type str) anymore?

Reply:
Yes we don't support type str address anymore, updated the respective testscripts in harness.